### DISCUSSION TITLE
Personalizzazione modulistica

### DISCUSSION THREAD

**Discussion by rodolfo024** [iisgiua/giuaschool#259] [Richieste di aiuto] [answered]
Come posso personalizzare la stampa del programma che viene scaricato in .docx?

**Comment by trinko** [comment, marked as answer]
Se intendi modificare come viene generato, devi necessariamente effettuare modifiche al codice della procedura.

Ma in realtà viene generato appositamente un .docx e non un PDF in modo che il docente possa modificare facilmente  quello che vuole.

**Reply by rodolfo024** [reply]
Ricordo bene quando personalizzai le varie modulistiche dello scrutinio ma per i programmi non so precisamente in quale parte di codice dover andare. Se me lo sapresti dire te ne sarei grato!

**Reply by trinko** [reply]
La generazione dei programmi svolti viene effettuata nel file PHP: **src/Controller/LezioniController.php**

La funzione si chiama **argomentiProgrammaAction** e utilizza la libreria PhpOffice per generare il documento Word.
